### DISCUSSION TITLE
Disappearing User - Next Steps?

### DISCUSSION THREAD

**Discussion by colinsham** [advplyr/audiobookshelf#1167] [Q&A] [answered]
So in addition to the default root user, I created a secondary user (just out of habit as good security). That's the one I use to connect using the Testflight iOS mobile app.

Sometime in the last week or so, this user has disappeared from ABS. It no longer exists in the list of users. The iOS app is still reporting playback from 'local' source as I was listening offline, but those listening sessions are attributed to user "N/A".

What should I do to fix this situation? I would like to maintain my user stats.

Can I just recreate the user with the same login name and all will be good?

**Comment by advplyr** [comment, marked as answer]
No, when you create a user it will have a new unique ID.
Have you checked the server logs? I'm not sure why a user would disappear, you should see some error logs.

**Reply by colinsham** [reply]
Okay for some reason this conversation made me take a closer look at the data store -> under /config/users/data - one of the data json files had a lock (folder) on it. I stopped the container, removed the lock, renamed, and then restarted. User is back and mobile app access works as well.

I uh, am unsure how to report this so that it could be improved, so I'm just going sneak away.
